main.go

```diff
@@ -193,15 +193,17 @@ func main() {
 			nextRefresh, err := time.Parse(time.RFC1123Z, lastUpdated)
 
 			if err != nil {
-				panic("Error while trying to parse date")
-			}
+				fmt.Printf("Error while trying to parse last update time")
```

Comment:
Use `Println` or add a trailing newline here, as the message would be immediately overwritten by other text in the current state.